### PR TITLE
feat: create azure-tools package and move script from dev-tools into it

### PR DESCRIPTION
**Pull request by matthieu-crouzet**
## Proposed change
create azure-tools package and move script from dev-tools into it

### CODE REVIEW

File: packages/@o3r/azure-tools/tsconfig.eslint.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../../tsconfig.jest",
+  "include": [
+    ".eslintrc.js",
+    "jest.config.js",
+    "src/**/*.spec.ts",
+    "src/helpers/**/*.ts",
```
Comment (kpanot) [inline packages/@o3r/azure-tools/tsconfig.eslint.json:7]:
```suggestion
```

File: packages/@o3r/azure-tools/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/* For IDE usage only */
+{
+  "extends": "../../../tsconfig.base",
+  "references": [
+    {
+      "path": "./tsconfig.build.json"
+    },
+    {
+      "path": "./tsconfig.eslint.json"
```
Comment (kpanot) [inline packages/@o3r/azure-tools/tsconfig.json:9]:
```suggestion
      "path": "./tsconfig.spec.json"
```

File: packages/@o3r/azure-tools/package.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@o3r/azure-tools",
```
Comment (kpanot) [inline packages/@o3r/azure-tools/package.json:2]: should be registered in the `tsconfig.base` and `tsconfig.build` at root level

**Review by kpanot — DISMISSED** [review]

File: packages/@o3r/azure-tools/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<h1 align="center">Otter Azure tools</h1>
+<p align="center">
+  <img src="https://raw.githubusercontent.com/AmadeusITGroup/otter/main/.attachments/otter.png" alt="Super cute Otter!" width="40%"/>
```
Comment (fpaul-1A) [inline packages/@o3r/azure-tools/README.md:3]: this won't work with https://github.com/AmadeusITGroup/otter/pull/874

File: packages/@o3r/azure-tools/README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+Various Azure DevOps tools
+
+## How to install
+
+This package can be used via `npx` command to executed on of the listed command line interfaces.
+
+```shell
+npx -p @o3r/azure-tools ...
+```
```
Comment (fpaul-1A) [inline packages/@o3r/azure-tools/README.md:16]: not sure this part is very helpful
it also seems confusing that the `How to install` section explains how to not install

File: packages/@o3r/azure-tools/README.md (hunk 4 above)
Comment (matthieu-crouzet) [reply packages/@o3r/azure-tools/README.md:3]: this is merged on main not on release 10-next for now

Comment (fpaul-1A) [reply packages/@o3r/azure-tools/README.md:3]: you are targeting `main` in the url

Comment (matthieu-crouzet) [reply packages/@o3r/azure-tools/README.md:3]: good point 😄

File: packages/@o3r/azure-tools/.compodocrc.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://raw.githubusercontent.com/compodoc/compodoc/develop/src/config/schema.json",
+  "name": "Azure tools",
+  "output": "../../../generated-doc/azure-tools",
+  "tsconfig": "./tsconfig.doc.json",
+  "assetsFolder": "../../../.attachments",
```
Comment (fpaul-1A) [inline packages/@o3r/azure-tools/.compodocrc.json:6]: we need to keep in mind that these needs to be changed when `main` will be cascaded to v10

**Review by fpaul-1A — APPROVED** [review]

**Review by mrednic-1A — DISMISSED** [review]

**Review by mrednic-1A — APPROVED** [review]

**Review by fpaul-1A — APPROVED** [review]

**Pull request merged**